README.md

```diff
@@ -10,36 +10,46 @@
 
 ![k0s logo](docs/img/k0s-logo-full-color.svg)
 
-k0s is an all-inclusive Kubernetes distribution with all the required bells and whistles preconfigured to make building a Kubernetes clusters a matter of just copying an executable to every host and running it.
+[k0s](https://k0sproject.io/) is an all-inclusive Kubernetes distribution, which is configured with all of the features needed to build a Kubernetes cluster and packaged as a single binary for ease of use.
+
+k0s fits well in any cloud environment, but can also be used in IoT gateways, Edge and Bare metal deployments due to its simple design, flexible deployment options and modest system requirements.
 
 ## Key Features
 
-- Packaged as a single static binary
-- Self-hosted, isolated control plane
-- Variety of storage backends: etcd, SQLite, MySQL (or any compatible), PostgreSQL
-- Elastic control-plane
-- Vanilla upstream Kubernetes
-- Supports custom container runtimes (containerd is the default)
-- Supports custom Container Network Interface (CNI) plugins (kube-router is the default)
+- Different installation methods: [single-node](docs/install.md), [multi-node](docs/k0sctl-install.md), [airgap](docs/airgap-install.md) and [Docker](docs/k0s-in-docker.md)
+- Automatic lifecycle management with k0sctl: [upgrade](docs/upgrade.md), [backup and restore](docs/backup.md)
+- Modest [system requirements](docs/system-requirements.md) (1 vCPU, 1 GB RAM)
+- Vanilla upstream Kubernetes (with no changes)
+- Available as a single binary with no [OS dependencies](docs/os-deps.md) besides the kernel
+- Flexible deployment options with [control plane isolation](docs/networking.md#controller-worker-communication)
```

Comment:
I would mention that it is a mode by default (which is not the case for the majority of k8s distros)